package.json

```diff
@@ -40,10 +41,10 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.93.0",
-    "@sentry/angular-ivy": "7.93.0",
-    "@sentry/react": "7.93.0",
-    "@sentry/vue": "7.93.0"
+    "@sentry/angular": "7.81.1",
+    "@sentry/angular-ivy": "7.81.1",
+    "@sentry/react": "7.81.1",
+    "@sentry/vue": "7.81.1"
```

Comment:
I see the downgrade is intentional to as all the packages from the JS monorepo must match in version. Should we follow up with JS upgrade PR? Before the next capacitor release?

Reply:
I would rather release a new version with only this change since it's blocking vue users from using it and on the next version, make the javascript bump.